regimes/it/charges.go

```diff
@@ -0,0 +1,25 @@
+package it
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/tax"
+)
+
+const (
+	ChargeTypeBollo cbc.Key = "bollo"
```

Comment:
It might be worth having the variable names in English.

```suggestion
	ChargeKeyStampDuty cbc.Key = "stamp-duty"
```